pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArithmeticFunctions.java

```diff
@@ -54,15 +54,27 @@ public static double mod(double a, double b) {
   }
 
   @ScalarFunction
-  public static double min(double a, double b) {
+  public static double least(double a, double b) {
     return Double.min(a, b);
   }
 
   @ScalarFunction
-  public static double max(double a, double b) {
+  public static double greatest(double a, double b) {
     return Double.max(a, b);
   }
 
+  @Deprecated
```

Comment:
We could just remove the deprecation and tolerate some duplication. What do you think?

Reply:
We should deprecate them as they are not standard sql. Let's add some release note in the PR description

Reply:
min/max were never actually possible to use in SQL because they clashed with the aggression function. Ingestion transforms aren't specified in SQL, so I'm not sure they really need to be SQL compliant.